pages/firststeps.md

```diff
@@ -25,37 +25,37 @@ A very large part of these steps is finding problems with these steps and this M
 There is no official deadline to complete these steps, but most candidates who are approved for the internship program finished the steps within 7-8 days. Good luck!
 
 
-##Step #0: Introductory Steps:
```

Comment:
As a matter of form rather than function, it's more a matter of opinion.  I find it a little redundant to label a number with the symbol for number, as anyone reading the First Steps likely has at least a moderate background in using numbers.  That being said, this is purely style rather than function, and I think it looks a little cleaner.  I will defer to whatever you guys feel looks better.